test/10-verify.spec.js

```diff
@@ -11,40 +11,50 @@ import jsigs from 'jsonld-signatures';
 import jsonld from 'jsonld';
 import {Ed25519Signature2018} from '@digitalbazaar/ed25519-signature-2018';
 import {CredentialIssuancePurpose} from '../lib/CredentialIssuancePurpose.js';
-
 import {mock as mockData} from './mocks/mock.data.js';
 import {v4 as uuid} from 'uuid';
 import * as vc from '../lib/index.js';
 import {MultiLoader} from './MultiLoader.js';
-import {contexts as realContexts} from '../lib/contexts/index.js';
 import {invalidContexts} from './contexts/index.js';
 import {credential as mockCredential} from './mocks/credential.js';
 import {assertionController} from './mocks/assertionController.js';
 import {VeresOneDriver} from 'did-veres-one';
+import {
+  CONTEXT_V1 as odrlCtx,
+  CONTEXT_URL_V1 as odrlCtxUrl
+} from '@digitalbazaar/odrl-context';
+import {
+  CONTEXT_V1 as vcExamplesV1Ctx,
+  CONTEXT_URL_V1 as vcExamplesV1CtxUrl
+} from '@digitalbazaar/credentials-examples-context';
 
-const contexts = Object.assign({}, realContexts);
-const testContextLoader = () => {
+const contexts = new Map();
+contexts.set(vcExamplesV1CtxUrl, vcExamplesV1Ctx);
+contexts.set(odrlCtxUrl, odrlCtx);
+const {extendContextLoader} = jsigs;
+const {defaultDocumentLoader} = vc;
+
+const testContextLoader = extendContextLoader(async url => {
   for(const key in invalidContexts) {
     const {url, value} = invalidContexts[key];
-    contexts[url] = value;
+    contexts.set(url, value);
   }
```

Comment:
This looks off to me ... won't this loop be called every time the document loader is used? Shouldn't it only be called once during some setup?

Reply:
You're right. I've moved the for in loop outside of the context loader. [b834a0b](https://github.com/digitalbazaar/vc/pull/117/commits/b834a0b708cea7d52fb24bd05e42813c0a7a2cf8)